src/includes/class-alma-wc-payment-validator.php

```diff
@@ -38,47 +38,71 @@ public static function validate_payment( $payment_id ) {
 		try {
 			$payment = $alma->payments->fetch( $payment_id );
 		} catch ( RequestError $e ) {
-			$logger->error( 'Error while fetching payment with id ' . $payment_id . ': ' . $e->getMessage() );
+			$logger->error( 'Error while fetching payment with id.', [
+                'Method' => __METHOD__ ,
```

Comment:
Can you add method in logger wrapper. ?

Reply:
it catch the current method, if i do it in the logger, it will be le logger method which be loaded